packages/wonder-blocks-modal/components/modal-backdrop.test.js

```diff
@@ -75,15 +75,18 @@ describe("ModalBackdrop", () => {
         expect(onCloseModal).not.toHaveBeenCalled();
     });
 
-    test("On mount, we focus the last button in the modal", () => {
+    test("On mount, we focus the last button in the modal", (done) => {
```

Comment:
Is it good or bad that as soon as I see `done` I wonder "should this by async?"? Doing so would make the arrange/act/assert easier to do.

Reply:
I can update this to use `async`/`await`.